pyplanet/utils/style.py

```diff
@@ -1,14 +1,12 @@
 import re
-import struct
 
-import binascii
 
-STRIP_ALL = dict(letters='wnoitsgz<>', part=r'\$[lh]\[.+\]|\$[lh]|\$[0-9a-f]{3}')
+STRIP_ALL = dict(letters='wnoitsgz<>', part=r'(?<!\$)\$[lh]\[.+\]|(?<!\$)\$[lh]|(?<!\$)\$[0-9a-f]{3}|(?<!\$)\$[f-y]{1}')
 """
 Strip all custom maniaplanet styles + formatting.
 """
 
-STRIP_COLORS = dict(letters='g', part=r'\$[0-9a-f]{3}')
+STRIP_COLORS = dict(letters='g', part=r'(?<!\$)\$[0-9a-f]{3}')
```

Comment:
I think we should change this from `{3}` to `{1,3}` to capture color codes that are 1 or 2 long. See other comment plus:
![image](https://github.com/PyPlanet/PyPlanet/assets/52106022/6c53ea56-dd3f-4c4c-a15a-a6e447551df1)